eng/pipelines/templates/steps/get-tagged-code.yml

```diff
@@ -0,0 +1,42 @@
+parameters:
+  - name: DestinationDirectory
+    type: string
+    default: ''
+  - name: Package
+    type: string
+    default: ''
+  - name: CheckoutPath
+    type: string
+    default: ''
+  - name: Version
+    type: string
+    default: ''
+          
+steps:
+  - pwsh: |
+      $targetPath = "$(Agent.TempDirectory)/${{ parameters.Package }}"
+      if (!(Test-Path $targetPath)) {
```

Comment:
you can also run `mkdir -p $targetPath` instead of checking the path's existence.